common/src/main/java/org/apache/flink/streaming/connectors/pulsar/internal/PulsarClientUtils.java

```diff
@@ -20,10 +20,12 @@
 import org.apache.pulsar.client.impl.conf.ClientConfigurationData;
 import org.apache.pulsar.shade.org.apache.commons.lang3.StringUtils;
 
+import java.util.Properties;
+
 /**
  * Utility to create Pulsar Admin Client from adminUrl and clientConfigurationData.
```

Comment:
change this also?

Reply:
After the batch read is merged, the configuration will be modified. Because the batch read contains a new configuration class